mix.exs

```diff
@@ -5,7 +5,7 @@ defmodule RemoteIp.Mixfile do
     [
       app: :remote_ip,
       version: "1.0.0",
-      elixir: "~> 1.7",
+      elixir: "~> 1.10",
```

Comment:
Cool!

Still to do: audit the deps. I can do that separately from this PR, but just making the note to myself. :)

Reply:
makes sense, for that could work to add [mix audit](https://hexdocs.pm/mix_audit) to the CI pipeline but I agree that is something that could go in another PR :)

Reply:
Not a bad idea, although `mix hex.audit` isn't quite what I had in mind. I think the sort of checking I do still requires some manual intervention.

Since I had a moment to check:
- combine is still at v0.10.0, so no need to bump it: https://hex.pm/packages/combine
- plug v1.8 is the earliest version still receiving security patches: https://github.com/elixir-plug/plug#supported-versions
- however, I'd previously bumped plug to ~> 1.10 because that was the earliest version to support the new minimum elixir version: https://github.com/ajvondrak/remote_ip/commit/27a108ffd89c60c8974afda68f8be24469cd9f14
- it looks like the earliest version of plug that required elixir ~> 1.10 is actually the latest one, v1.14 😱 https://github.com/elixir-plug/plug/blob/main/CHANGELOG.md#v1140-2022-10-31
- so I guess we should bump the plug dependency to ~> 1.14 if we're targeting elixir 1.10+; makes me a bit nervous, but hopefully people aren't hard-locked to a minor version for some reason 😕 

The dev/ci dependencies look like they might be experiencing some errors of their own in CI. It's less important that they have the minimum possible versions, but I would perform a similar pass on those.

I can go ahead and merge this without you having to figure out those CI failures, though, unless you want to take a crack at them (and wait for me to hit the button so CI runs 😅). Seems like the sort of work that will just be mashed into the rest of the dependency bumps.

Reply:
Oh, literally did not read the thing you linked to. The mix_audit package looks pretty sweet! 😀 I'll have to check it out some more.